tools/ossf_best_practices/passing_criteria.md

```diff
@@ -0,0 +1,397 @@
+
+**Met**
+https://github.com/nodejs/node/issues
+
+> The project SHOULD use an issue tracker for tracking individual issues.
+
+**Met**
+https://github.com/nodejs/node/issues
+
+> The project MUST acknowledge a majority of bug reports submitted in the last 2-12 months (inclusive); the response need not include a fix.
+
+**Met**
+
+> The project SHOULD respond to a majority (>50%) of enhancement requests in the last 2-12 months (inclusive). 
+
+**Met**
+
+> The project MUST have a publicly available archive for reports and responses for later searching. (URL required)
+
+**Met**
+https://github.com/nodejs/node/issues
+
+
+## Vulnerability report process
+
+> The project MUST publish the process for reporting vulnerabilities on the project site. (URL required)
+**Met** https://nodejs.org/en/security/
+
+> If private vulnerability reports are supported, the project MUST include how to send the information in a way that is kept private. (URL required)
+
+**Met** https://nodejs.org/en/security/#disclosure-policy
+
+> The project's initial response time for any vulnerability report received in the last 6 months MUST be less than or equal to 14 days.
+
+**Met** Your email will be acknowledged within 24 hours
+
+
+# Quality
+
+## Working build system
+
+> If the software produced by the project requires building for use, the project MUST provide a working build system that can automatically rebuild the software from source code.
+
+**Met**
+https://github.com/nodejs/node#build
+
+
+> It is SUGGESTED that common tools be used for building the software.
+
+**Met**
+python, make or batch file, gcc or clang
+
+
+> The project SHOULD be buildable using only FLOSS tools.
+
+**Met**
+python, make or batch file, gcc or clang
+
+
+## Automated test suite
+
+> The project MUST use at least one automated test suite that is publicly released as FLOSS (this test suite may be maintained as a separate FLOSS project). The project MUST clearly show or document how to run the test suite(s) (e.g., via a continuous integration (CI) script or via documentation in files such as BUILD.md, README.md, or CONTRIBUTING.md).
+
+**Met**
+
+
+> A test suite SHOULD be invocable in a standard way for that language.
+
+**Met**
+make or batch file, executed using python
+
+
+> It is SUGGESTED that the test suite cover most (or ideally all) the code branches, input fields, and functionality.
+
+**Met**
+
+
+> It is SUGGESTED that the project implement continuous integration (where new or changed code is frequently integrated into a central code repository and automated tests are run on the result).
+
+**Met**
+https://ci.nodejs.org/
+
+
+## New functionality testing
+
+> The project MUST have a general policy (formal or not) that as major new functionality is added to the software produced by the project, tests of that functionality should be added to an automated test suite.
+
+**Met** 
+https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#step-5-test
```

Comment:
This link needs an update.

Reply:
Solved in [3f18137](https://github.com/nodejs/security-wg/pull/954/commits/3f18137c8f4fa507612f1f006353095963bd0991)